src/v/utils/human.h

```diff
@@ -23,8 +24,13 @@ struct bytes {
     friend std::ostream& operator<<(std::ostream& o, const bytes&);
 };
 struct latency {
+    // the input should be in microseconds
     explicit latency(double x)
       : value(x) {}
+    template<typename Duration>
+    explicit latency(Duration d)
```

Comment:
i can't ever remember. would `latency((double)0.0)` resolve to the template overload or the more specific overload with the double type? if the later, i guess we checked all the call sites that they were already passing in micros?

Reply:
https://godbolt.org/z/hq6nrTG4j

There is only a single callsite and yes it's passing in explicit doubles as macros